catfs/db/database_badger.go

```diff
@@ -323,6 +400,10 @@ func (db *BadgerDatabase) Flush() error {
 
 	db.txn = nil
 	db.haveWrites = false
+	if time.Now().After(db.gcNeededAt) && !db.needsGC {
+		db.gcNeededAt = time.Now().Add(120 * time.Second) // badger updates stats once per minute, we give ample time for stats to settle
```

Comment:
Just a style thing: Please put the comments above the line they're referring to to avoid extra long lines.

Reply:
done